tests/globalkilltest/run-tests.sh

```diff
@@ -14,7 +14,7 @@
 # limitations under the License.
 
 set -eu
-trap 'set +e; PIDS=$(jobs -p); [ -n "$PIDS" ] && kill -9 $PIDS' EXIT
```

Comment:
we don't need to add a `for` loop? I think the key  point of this fix is `|| true`

Reply:
That is better will fix it on weekend~

On vocation for now 😌

Reply:
@lance6716 checked kill -9 xxx || true  also have this error
we can only kill one by one then || true

Reply:
I did a simple test

```
[root@vm172-16-201-151 cse]# pgrep vim
473188
[root@vm172-16-201-151 cse]# kill 12345 473188 || true && echo "$?"
-bash: kill: (12345) - No such process
0
```

seems we don't need to kill them one by one. And this is my bash version

```
[root@vm172-16-201-151 cse]# bash --version
GNU bash, version 4.4.20(1)-release (x86_64-redhat-linux-gnu)
Copyright (C) 2016 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>

This is free software; you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.
```